src/components/cc-env-var-input/cc-env-var-input.js

```diff
@@ -127,19 +127,20 @@ export class CcEnvVarInput extends LitElement {
       // language=CSS
       css`
         :host {
-          --cc-gap: 0.5rem;
+          --cc-gap: 0.5em;
           display: block;
         }
 
         .name {
           box-sizing: border-box;
           display: inline-block;
-          flex: 1 1 15rem;
+          flex: 1 1 17.14em;
           font-family: var(--cc-ff-monospace, monospace);
           /* I have a bug on Linux between Chrome and FF with rem on inputs */
```

Comment:
@hsablonniere I don't know what bug it was but its weird because this selector is related to a `<span>` and inputs are not impacted by it.

Do you remember what bug it was and do you know if the `px` font-size is still necessary ? If not i'll just remove my TODO comment :wink:

Reply:
I cannot reproduce the bug, I think it's OK now. :+1:

Reply:
Removed the comment, kept the font size in `em` and changed `17.14` to `17` :+1: